app/styles/mobile.scss

```diff
@@ -103,11 +103,11 @@
 #footer {
   padding-top: 50px;
   text-align: left;
-  a{
-    color:black;
+  a:hover{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space